app/code/core/Mage/Core/Helper/Data.php

```diff
@@ -200,7 +199,7 @@ public function formatTime($time = null, $format = Mage_Core_Model_Locale::FORMA
 
         if (is_null($time)) {
             $date = Mage::app()->getLocale()->date(time());
-        } else if ($time instanceof Zend_Date) {
+        } elseif ($time instanceof Zend_Date) {
```

Comment:
Difference `else if` and `elseif`:

> Note: Note that elseif and else if will only be considered exactly the same when using curly brackets as in the above example. When using a colon to define your if/elseif conditions, you must not separate else if into two words, or PHP will fail with a parse error.

#ref https://www.php.net/manual/en/control-structures.elseif.php

Reply:
Since we have curly brackets it doesnt matter?